src/plugins/base.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, bot):
         """
         self.bot = bot
 
-    def handle_tokens(self, channel, msg, keywords, callback):
```

Comment:
Oh, we don't need to move chanels at all, but I would actually preffer to move all callback args to *args so that the callback function won't need any unused parameters that pylint will complain about